readme.md

```diff
@@ -125,6 +125,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 	- [Critical-Path (Above-the-fold) Tools](https://github.com/addyosmani/critical-path-css-tools)
 	- [Scalability](https://github.com/davidtheclark/scalable-css-reading-list)
 	- [Must-Watch Talks](https://github.com/AllThingsSmitty/must-watch-css)
+    - [CSS Protips](https://github.com/AllThingsSmitty/css-protips)
```

Comment:
Thanks for the suggestion :+1: but could you change the title simply to `Protips`? It's already implied that they're CSS Protips as it's under the CSS bullet point.